base/show.jl

```diff
@@ -595,7 +595,11 @@ function show(io::IO, p::Pair)
     for i = (1, 2)
         io_i = IOContext(iocompact, :typeinfo => typeinfos[i])
         isdelimited(io_i, p[i]) || print(io, "(")
-        show(io_i, p[i])
+        if i == 1 && isoperator(Symbol(p[i]))
+           print(io_i, "(", p[i], ")")
```

Comment:
Incorrect indentation here: should be 4 spaces, not 3.

Reply:
Thanks @ararslan and @fredrikekre, I added the changes.